app/i18n/locales/en/programs-page.json

```diff
@@ -66,9 +66,9 @@
     ]
   },
   "acceleratorProgram": {
```

Comment:
Might be cleaner to rename variables in the code too?

Reply:
@kalidiagne Please~ :)